### PR TITLE
Adding support for stop loss

### PR DESCRIPTION
**Pull request by dluco-**

### CODE REVIEW

File: avanza/avanza.py [hunk 1]
```diff
@@ -1670,6 +1671,67 @@ def place_order_sell_fund(
         )
 
 
+    def place_stop_loss_order(
+        self,
+        parent_stop_loss_id: str,
+        account_id: str,
+        order_book_id: str,
+        stop_loss_trigger: Type[StopLossTrigger],
+        stop_loss_order_event: Type[StopLossOrderEvent],
```
Comment (Qluxzz) [inline avanza/avanza.py:1680]: Shouldn't these just be StopLossTrigger and StopLossOrderEvent, without being wrapped in a `Type[]`?

https://docs.python.org/3/library/typing.html#typing.Type

**Review by Qluxzz — CHANGES_REQUESTED** [review]

File: avanza/avanza.py (hunk 1 above)
Comment (dluco-) [reply avanza/avanza.py:1680]: Probably should, good catch. I'll update the PR.

**Comment by dluco-** [comment]
Anything else to add/change?

**Comment by Qluxzz** [comment]
No, it LGTM. Thanks for contributing!

**Pull request merged**